app/views/export/generic.html.erb

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <style>
+    .divider:nth-child(odd) {
+      border-top: thick double;
+      border-bottom: thick double;
+      background-color: #e3e3e3;
+    }
+  </style>
+  <title><%= "test" %></title>
+</head>
+<body>
+<% id_count = nil %>
+<% @data.each do |data| %>
+  <% if id_count == nil %>
+    <% id_count = data[0].to_i.truncate %>
+    <div class="divider">
+  <% end %>
+  <% if id_count != data[0].to_i.truncate %>
+    </div><div class="divider">
+    <% id_count = data[0].to_i.truncate %>
+  <% end %>
+  <p><%= data[0] %> - <%= raw data[1] %>: <span style="color:#137300"><%= data[2] %></span></p>
```

Comment:
One aspect that made the old CSV harder to read was that a bunch of "answers" had HTML within them, so now we can render them fully.  This has the advantage of allowing href links to be clickable (although some of them might point to the app itself, which obviously isn't available), allow basic styling, and all the other goodness that HTML rendering provides.

Reply:
you might want to `raw data[2]` as well? I've seem some escaped html here

Reply:
Yeah, I don't see any harm in doing that there too.  There's likely certain answers that likely have HTML in it.